AppCenter/AppCenterTests/MSAuthTokenContextTests.m

```diff
@@ -34,15 +34,45 @@ - (void)testSetAuthToken {
 
   // If
```

Comment:
You should divide into more tests for more readability. This test should stay very basic, no previous token/account then new token/account: both callback called. 

Second test: Already a token/account then new token/account: both callbacks triggered.

Third test: Already a token/account then same token but different account: only didReceiveAuthToken triggered.